bark/models/behavior/motion_primitives/macro_actions.cpp

```diff
@@ -77,8 +77,7 @@ BehaviorMPMacroActions::GetMotionPrimitives() const {
 BehaviorMotionPrimitives::MotionIdx
 BehaviorMPMacroActions::GetNumMotionPrimitives(
     const ObservedWorldPtr& observed_world) {
-  return GetNumMotionPrimitivesByCorridors(*observed_world,
```

Comment:
Otherwise there is missing a function which gives you the number of all available motion primites and one function is only a wrapper of another one. 
If one needs the current available motion primitives GetNumMotionPrimitivesByCorridors(*observed_world) can be called. I could change the interface to GetNumMotionPrimitives() without passing observed world.

Reply:
okay